bimsb/packages/staging.scm

```diff
@@ -3024,3 +3024,25 @@ the xlxs package.  This release corresponds to POI 3.10.1.")
        "MuSiC is a deconvolution method that utilizes cross-subject scRNA-seq
 to estimate cell type proportions in bulk RNA-seq data.")
       (license license:gpl3))))
+
+(define-public r-dsa
+  (let ((commit "f181f549fd86feb58955f11e53ee38da67ac4f5e"))
+    (package
+      (name "r-dsa")
+      (version "1.0")
```

Comment:
Here I'd do `(version (git-version "1.0" revision commit))`.

This repository has no tags, so the version might still nominally be `1.0` even on a later commit.  Since commit hashes are random and thus cannot simply be ordered we need to use a monotonically increasing `revision` string to impose an ordering for versions.